src/js/components/Calendar/__tests__/Calendar-test.tsx

```diff
@@ -38,6 +38,17 @@ describe('Calendar', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('date without time specified', () => {
+    // correct date displayed if time is not specified in ISOstring
+    const { container } = render(
+      <Grommet>
+        <Calendar date={DATE.split('T')[0]} animate={false} />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
I think we can just use an assertion here to verify we have the expected date instead of needing to use a snapshot

Reply:
Hello, i updated the test to not use snapshot.